modules/docs/mdx/CONTRIBUTING.mdx

```diff
@@ -54,38 +49,25 @@ If you are contributing code, please take a look at the following sections to fa
 with how the Canvas Kit repo is organized and run. This will help streamline the pull request
 process. If you have any questions, please reach out!
 
-If your contribution is visual or UI-based, please ensure you consult with a designer so the
-contribution can be evaluated as an end-to-end process.
-
-#### Automation on Commit
-
-Upon commit, [lint-staged](https://github.com/okonet/lint-staged) will run your staged code through
-[Prettier](https://prettier.io) and [eslint](https://eslint.org).
-
-#### Storybook
-
-Canvas Kit uses [Storybook](https://storybook.js.org/) for the component development environment.
-
-#### Yarn and Workspaces
-
-Canvas Kit uses Yarn for package management and takes advantage of its support for
-[Workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) to connect all of its different modules
-within a single repository.
-
-#### Testing
+If your contribution is visual or UI-based, please ensure you consult with a designer or request a
```

Comment:
This didn't move but I edited the content slightly to accommodate external contributors.